integration_tests/e2e/goal/createGoals.cy.ts

```diff
@@ -0,0 +1,57 @@
+import Page from '../../pages/page'
+import CreateGoalsPage from '../../pages/goal/CreateGoalsPage'
+import OverviewPage from '../../pages/overview/OverviewPage'
+
+context('Create goals', () => {
```

Comment:
This is the new cypress test for the new create goals screen.
At the moment it only has 2 scenarios because that is as much of the functionality that we've written this far:
1. Navigate to the page directly via its URL
1. Click to create a Goal from the Overview page, and partially complete the form, triggering a validation error